tests/functionnal/Glpi/Inventory/Assets/Computer.php

```diff
@@ -35,6 +35,8 @@
 
 namespace tests\units\Glpi\Inventory\Asset;
 
+use Agent;
+
```

Comment:
```suggestion
```

Not usefull since you use `\Agent` in code (which is the way it's done for tests most of the time).